client/trino-jdbc/src/main/java/io/trino/jdbc/ConnectionProperties.java

```diff
@@ -645,6 +650,15 @@ public ResolverContext()
         }
     }
 
+    private static class HostnameInCertificate
+            extends AbstractConnectionProperty<String>
+    {
+        public HostnameInCertificate()
+        {
+            super("hostnameInCertificate", NOT_REQUIRED, SslVerification.IF_FULL_SSL_VERIFICATION_ENABLED, STRING_CONVERTER);
```

Comment:
nit static import for IF_FULL_SSL_VERIFICATION_ENABLED

Reply:
This was consistent with other imports in the file!